_overviews/scala3-book/types-intersection.md

```diff
@@ -13,6 +13,9 @@ Used on types, the `&` operator creates a so called _intersection type_.
 The type `A & B` represents values that are **both** of the type `A` and of the type `B` at the same time.
 For instance, the following example uses the intersection type `Resettable & Growable[String]`:
 
+{% tabs reset-grow class=tabs-scala-version %}
```

Comment:
When the file is Scala 3 only, you can add this at the beginning of the file:

```html
<span class="tag tag-inline">Scala 3 Only</span>
```

Reply:
done